cmd/drone-server/server.go

```diff
@@ -89,13 +90,30 @@ func server(c *cli.Context) error {
 	store_ := setupStore(c)
 	setupEvilGlobals(c, store_, remote_)
 
-	// we are switching from gin to httpservermux|treemux,
```

Comment:
... // we are switching from gin to httpservermux|treemux, ...

please do not remove - I think we should convert it into an TODO ...

Reply:
I don't think it is a TODO, it sound more like a we are switching context from gin to httpservermux|treemux to me.

Reply:
to be hosntest - it's guly and we only should use one router - so use gin for all and if it does not support all there are other drop in replacements

lets leave it out I'll create an issue!